packages/ecdh/tests/ecdh.test.ts

```diff
@@ -0,0 +1,57 @@
+import { BarretenbergBackend } from '@noir-lang/backend_barretenberg';
+import { Noir } from '@noir-lang/noir_js';
+import { ProofData } from '@noir-lang/types';
+import { expect } from 'chai';
+import { randomBytes } from 'crypto';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import 'mocha';
+import circuit_ecdh from '../../../target/ecdh.json';
+
+function generatePrivateKey(): Uint8Array {
+  return randomBytes(32);
+}
+
+describe('ECDH Circuit Tests', function() {
+  let noir: Noir;
+  let backend: BarretenbergBackend;
+  let correctProof: ProofData;
+
+  beforeEach(async () => {
+    const circuitFile = readFileSync(resolve(__dirname, '../../../target/ecdh.json'), 'utf-8');
+    const circuit = JSON.parse(circuitFile);
+    backend = new BarretenbergBackend(circuit);
+    noir = new Noir(circuit, backend);
+  });
+
+  it('Should generate valid proof for correct input', async function() {
+    this.timeout(20000); // Optional: Increase timeout if needed
```

Comment:
let's increase it a bit
(I had a timeout error locally)
```suggestion
    this.timeout(25000)
```